docs/tutorial/rust/src/creating_the_tiles_from_rust.md

```diff
@@ -1,12 +1,10 @@
 # Creating The Tiles From Rust
 
 The tiles in the game should have a random placement. We'll need to add the `rand` dependency to
-`Cargo.toml` for the randomization.
+`Cargo.toml` for the randomization, using the `cargo` command.
 
-```toml
-[dependencies]
-slint = "0.3.4"
-rand = "0.8" # Added
+```sh
+cargo add rand
```

Comment:
Excellent idea

Reply:
Thinking about it, there is a risk that someone reading the tutorial many years from now (and because we will forget to update it) would import rand 0.9 or 1.0 which may have change its API that we use.
Not so likely though